format/format.go

```diff
@@ -1154,3 +1188,19 @@ func (w *writer) down() {
 	}
 	w.level--
 }
+
+func ensureFutureKeywordImport(imps []*ast.Import, kw string) []*ast.Import {
+	kwPath := ast.MustParseTerm("future.keywords." + kw)
+	for _, imp := range imps {
+		path := imp.Path.Value.(ast.Ref)
+
+		if ast.FutureRootDocument.Equal(path[0]) &&
+			string(path[1].Value.(ast.String)) == "keywords" {
```

Comment:
nit: do we need a len(path) == 2 check on this? Perhaps just declare the catch-all `future.keywords` ref and compare it here?